examples/mnasnet/mnasnet.py

```diff
@@ -0,0 +1,324 @@
+        self.class_names = ['airplanes', 'cars', 'birds', 'cats', 'deer', 'dogs', 'frogs', 'horses', 'ships', 'trucks']
+    
+    def _init_val_set(self):
+        self.val_set = vision.datasets.CIFAR10(root='./data', train=False,
+                            download=True, transform=self.transform)
+    
+    def _init_test_set(self):
+        self.test_set = vision.datasets.CIFAR10(root='./data', train=False,
+                            download=True, transform=self.transform)
+
+    def train_dataloader(self):
+        kwargs = dict(
+            batch_size=self.batch_size,
+            shuffle=True,
+            num_workers=2
+        )
+        self.train_loader = torch.utils.data.DataLoader(self.train_set, **kwargs)
+        return self.train_loader
+    
+    def val_dataloader(self):
+        kwargs = dict(
+            batch_size=self.batch_size,
+            shuffle=True,
+            num_workers=2
+        )
+        self.val_loader = torch.utils.data.DataLoader(self.val_set, **kwargs)
+        return self.val_loader
+    
+    def test_dataloader(self, batch_size):
+        kwargs = dict(
+            batch_size=batch_size,
+            shuffle=True,
+            num_workers=2
+        )
+        self.test_loader = torch.utils.data.DataLoader(self.val_set, **kwargs)
+        return self.test_loader
+
+@METRICS.register()
+class MyClassificationMetrics(pl.metrics.Metric):
+    def __init__(self, num_classes, *, acc={}, f1={}, prec={}, rec={}):
+        super().__init__()
+        prec.update(dict(num_classes=num_classes))
+        rec.update(dict(num_classes=num_classes))
+        f1.update(dict(num_classes=num_classes))
+        self.metrics = nn.ModuleList([
+            pl.metrics.Accuracy(**acc),
+            pl.metrics.F1(**f1),
+            pl.metrics.Precision(**prec),
+            pl.metrics.Recall(**rec),
+        ])
+        self.metrics_args = dict(
+            num_classes=num_classes,
+            acc=acc, f1=f1, prec=prec, rec=rec
+        )
+
+    def update(self, inputs, targets):
+        if isinstance(inputs, list):
+            inputs = np.concatenate([inp['class_label'] for inp in inputs])
+            inputs = torch.as_tensor(inputs).to(targets.device)
+        for metric in self.metrics:
+            metric.update(inputs, targets)
+    
+    def compute(self):
+        results = {}
+        typename = lambda x: type(x).__name__.split('.')[-1]
+        for metric in self.metrics:
+            name = typename(metric)
+            results[name] = metric.compute()
+        return results
+
+# %%
+# 2. Preparing the Model
+# ----------------------
+# `vortex`'s `ModelBase` is just an extension to `pl.LightningModule`,
+# so we can use it just like any `LightningModule`, like define
+# `training_step`, `validation_step`, `configure_optimizers`, etc.
+# 
+# Additionally, we need to define the following methods:
+# - `input_names` should return a list of string representing the input names
```

Comment:
you need indentation here to make the bulleted point work. also the inline code is double ticks.

Reply:
ok

Reply:
Aside from the double tick, I don't think it's the indentation though, I tried and it didn't work. I believe the format is valid rst since I tried it using different renderer. So i think this is sphinx or sphinx-gallery thing.

Let me change to table instead

Reply:
I've actually tried this, and it works, you need to use 3 space since the content starts after the first space, like this:

```
# Additionally, we need to define the following methods:
#   - `input_names` should return a list of string representing the input names
```

Reply:
hmm I already changed to table